Jenkinsfile

```diff
@@ -0,0 +1,180 @@
+ * works of the software solely for use with RTI Connext DDS. Licensee may
+ * redistribute copies of the software provided that all such copies are subject
+ * to this license. The software is provided "as is", with no warranty of any
+ * type, including any warranty for fitness for any purpose. RTI is under no
+ * obligation to maintain or support the software. RTI shall not be liable for
+ * any incidental or consequential damages arising out of the use or inability
+ * to use the software.
+ */
+def DETAILS_URL="https://community.rti.com/"
+def detailsText
+def detailsTextBuild
+def detailstextStatic
+
+pipeline {
+    agent none
+
+    stages {
+        stage('Executor Check') {
+            steps {
+                publishChecks detailsURL: DETAILS_URL, name: 'Waiting for executor',
+                        status: 'IN_PROGRESS', title: 'Waiting',
+                        summary: ':hourglass: Waiting for next available executor...'
+            }
+        }
+
+        stage('Build sequence') {
+            agent {
+                dockerfile {
+                    filename 'Dockerfile'
+                    label 'docker'
+                }
+            }
+
+            environment {
+                RTI_INSTALLATION_PATH = "${WORKSPACE}/unlicensed"
+                RTI_LOGS_FILE = "${WORKSPACE}/output_logs.txt"
+            }
+
+            stages {
+                stage('Download Packages') {
+                    steps {
+                        publishChecks detailsURL: DETAILS_URL, name: 'Waiting for executor',
+                                summary: ':white_check_mark: Build started.',
+                                title: 'Passed'
+
+                        sh 'python3 resources/ci_cd/jenkins_output.py'
+
+                        script {
+                            detailsText = readFile("jenkins_output.md")
+                        }
+
+                        publishChecks detailsURL: DETAILS_URL, name: STAGE_NAME,
+                            status: 'IN_PROGRESS', title: 'Downloading', text: detailsText,
+                            summary: ':arrow_down: Downloading RTI Connext DDS libraries...'
+
+                        rtDownload (
+                            serverId: 'rti-artifactory',
+                            spec: '''{
+                                "files": [
+                                {
+                                    "pattern": "connext-ci/pro/weekly/",
+                                    "props": "rti.artifact.architecture=x64Linux4gcc7.3.0;rti.artifact.kind=staging",
+                                    "sortBy": ["created"],
+                                    "sortOrder": "desc",
+                                    "limit": 1,
+                                    "flat": true
+                                }]
+                            }''',
+                        )
+
+                        sh 'tar zxvf connextdds-staging-x64Linux4gcc7.3.0.tgz unlicensed/'
```

Comment:
Add a reference to https://www.jfrog.com/jira/browse/HAP-1154, that is the reason why we cannot use the `explode` functionality